packages/twenty-cli/src/utils/app-manifest-loader.ts

```diff
@@ -121,6 +122,44 @@ export const loadManifest = async (
 
   const rawYarnLock = await fs.readFile(yarnLockPath, 'utf8');
 
+  const agents = await loadCoreEntity(
+    path.join(appPath, 'agents'),
+    (manifest, path) => validateSchema('agent', manifest, path),
+  );
+
+  const objectFromManifests = await loadCoreEntity(
+    path.join(appPath, 'objects'),
+    (manifest, path) => validateSchema('object', manifest, path),
+  );
+
+  const serverlessFunctionsFromManifest = await loadCoreEntity(
+    path.join(appPath, 'serverlessFunctions'),
+    (manifest, path) => validateSchema('serverlessFunction', manifest, path),
+  );
```

Comment:
Would it make sense to load these three core entities concurrently with a `Promise.all`?